cg/models/cg_config.py

```diff
@@ -633,6 +634,17 @@ def pdc_service(self) -> PdcService:
             self.pdc_service_ = service
         return service
 
+    @property
+    def run_directory_names_services(self) -> RunDirectoryNamesServices:
+        services = self.__dict__.get("run_directory_names_services_")
```

Comment:
What is the difference between this and `self.run_directory_names_services_`?

Reply:
Does it crash with my suggestion? Otherwise I would probably change it

Reply:
changed